Python/Default.sublime-keymap

```diff
@@ -0,0 +1,24 @@
+[
+    // Auto-pair quotes even after string modifiers.
+    // Copied over from the default bindings with moficiations to `preceding_text`.
+    { "keys": ["\""], "command": "insert_snippet", "args": {"contents": "\"$0\""}, "context":
+        [
+            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+            { "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|\\}|>|$)", "match_all": true },
+            { "key": "preceding_text", "operator": "regex_contains", "operand": "^$|[^\"a-zA-Z0-9_]$|(?i)\b[bfru]+$", "match_all": true },
```

Comment:
Looks like the `\b` needs to be a `\\b` here to be applied as word-boundary. Furthermore I am wondering which function `^$` has? Works without it as well.

```
{ "key": "preceding_text", "operator": "regex_contains", "operand": "[^\"a-zA-Z0-9_]$|(?i)\\b[bfru]+$", "match_all": true },
```

Reply:
:man_facepalming: JSON gets me every time.

The `^$` is needed for the beginning of the line because there's no character to be matched in the second pattern.

Reply:
Works fine without `^$` at beginning of a line or even at the very beginning of a file. Therefore I ask.

Reply:
So, i can't test this right now, but do you have any idea why? I'm not seeing a reason for it to work without ^$.

Reply:
Looks like ST's default key binding applies at the beginning of a line with or without `^$` in Python's context.

Reply:
Right, I only need to apply the special bindings for the special case as the default bindings will still be active.

Fixed with 5951a1a8a9a9880424e4d586a72e810d9dd7e3cc.